src/test/com/test/GetFilteredTweets_Test.java

```diff
@@ -0,0 +1,122 @@
+package com.test;
+
+import UTwitter.resources.Controller;
+import UTwitter.resources.MessageRequest;
+import UTwitter.service.TwitterImplement;
+import model.TwitterResponseModel;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import twitter4j.*;
+
+import javax.ws.rs.core.Response;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GetFilteredTweets_Test {
```

Comment:
File names usually doesn't have underscores so please check with 
Naming conversion link that I had sent in channel